app/Http/Controllers/AuthController.php

```diff
@@ -40,6 +40,18 @@ public function loginAction(LoginRequest $request): JsonResponse
         if (is_null($me->password)) {
             throw new FailedLoginException();
         }
+        $invites = ProjectInvite::where('email_address', $me->email_address)->get();
+        foreach ($invites as $invite) {
+            $me->projects()->sync([$invite->project_id => ['is_monitoring' => true]]);
+            if ($me->organisation_id == null) {
+                $me->organisation_id = $invite->project->organisation_id;
```

Comment:
Should this really be setting the the org to the last one from the list of invites? This code feels like something that isn't supposed to be in a loop; are we expecting there to be multiple invites for a user? If not, perhaps this should throw an error if there are multiple instead of what it's doing here.

Reply:
A similar situation occurred once there was more than one invitation for a user and that is why this logic was implemented, in fact it is similar to the "verifyUnauthorizedAction" function found in "AuthController.php" but this time it is being used at login.